bitnami/ghost/README.md

```diff
@@ -214,6 +214,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `persistence.accessModes`                     | Persistent Volume access modes                                                                  | `[]`                    |
 | `persistence.size`                            | Persistent Volume size                                                                          | `8Gi`                   |
 | `persistence.existingClaim`                   | The name of an existing PVC to use for persistence                                              | `""`                    |
+| `persistence.existingMountSubPath`                   | The name of an existing volume's sub path to mount for persistence                                              | `""`                    |
```

Comment:
Please use the ReadmeGenerator to update this: https://github.com/bitnami-labs/readme-generator-for-helm